projects/github-actions/test-results-to-slack/src/slack.js

```diff
@@ -52,6 +51,7 @@ async function postOrUpdateMessage( client, update, options ) {
 				} );
```

Comment:
If you're adding error checking everywhere else, why not check `response.ok` here too?

Reply:
Good point. Done in 541249a59607c48c03a281d53e49ee6bf13e9e98